src/main/java/me/zeroeightsix/kami/util/astaralg/Tunnels.java

```diff
@@ -0,0 +1,89 @@
+package me.zeroeightsix.kami.util.astaralg;
+
+import net.minecraft.util.math.Vec3i;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by fred41 on 18/07/2020.
+ */
+
+public class Tunnels {
+    public static final List<Vec3i> XTUNNEL = new ArrayList<Vec3i>() {{
+        add(new Vec3i(2, 0, 0));
+        add(new Vec3i(1, 0, 0));
+        add(new Vec3i(0, 0, 0));
+        add(new Vec3i(-1, 0, 0));
+        add(new Vec3i(-2, 0, 0));
+        add(new Vec3i(2, 1, 0));
+        add(new Vec3i(1, 1, 0));
+        add(new Vec3i(0, 1, 0));
+        add(new Vec3i(-1, 1, 0));
+        add(new Vec3i(-2, 1, 0));
+    }};
+    public static final List<Vec3i> ZTUNNEL = new ArrayList<Vec3i>() {{
+        add(new Vec3i(0, 0, 2));
+        add(new Vec3i(0, 0, 1));
+        add(new Vec3i(0, 0, 0));
+        add(new Vec3i(0, 0, -1));
+        add(new Vec3i(0, 0, -2));
+        add(new Vec3i(0, 1, 2));
+        add(new Vec3i(0, 1, 1));
+        add(new Vec3i(0, 1, 0));
+        add(new Vec3i(0, 1, -1));
+        add(new Vec3i(0, 1, -2));
+    }};
+    public static final List<Vec3i> PPNNTUNNEL = new ArrayList<Vec3i>() {{
```

Comment:
What does this name mean?

Reply:
i use the same tunnel for both directions on every line P= positive N= negative -> this is the tunnel for ++ and --  highways

Reply:
let me know if you have a better naming idea

Reply:
Could be more clear, maybe have a comment or name it something more intuitive, like Plus_Neg_Axis